.github/workflows/check-semver.yml

```diff
@@ -9,12 +9,39 @@ on:
       - labeled
       - unlabeled
 
+permissions:
+  issues: read
+  pull-requests: read
```

Comment:
This limits the permissions needed for this action - this is considered good practice